styleSheet.css

```diff
@@ -228,14 +229,73 @@ ul {
   margin: 5% 6px 0% 6px;
 }
 
-.network__icon {
-  width: calc(0.6em + 1.2vw);
+.network__icon{
+  width: calc(0.6rem + 0.2vw + 1.6vh);
 }
 
 .network__icon > img {
   width: 100%;
 }
 
+#contact {
+  background-image: url(images/contactSection_mail1.png);
+  height: calc(30rem + 15vh + 15vw);
+}
+
+.contact__elementBackground {
+  background-image: linear-gradient(rgba(50,50,50,0), var(--primary__color) 29%, var(--background__color));
+  left: 50%;
+  padding: 25px;
+  position: absolute;
+  text-align: center;
+  transform: translateX(-50%);
+  top: 15%;
+  width: 44%;
+}
+
+form {
+  display: grid;
+  grid-auto-columns: 20rem 20rem;
+}
+
+label {
+  display: none;
+}
+
+input {
+  font-size: 1rem;
+  height: 2rem;
+  width: 85%;
+}
+
+textarea {
+  font-size: 1rem;
+  width: 85%;
+}
+
+@media only screen and (max-width: 1366px)  {
+  
+  #contact {
+    height: calc(30rem + 20vh + 20vw);
+  }
+
+  .contact__elementBackground {
+    left: 8%;
+    top:  5%;
+    transform: none;
+    width: 80%;
+  }
+
+  .contact__formPhone, .contact__formSubject {
+    width: unset !important;
+  }
+  
+  form {
```

Comment:
be careful using selectors like this, because if you had many forms this could produce undesired style changes. Always prefer using classes